src/server/game/Spells/Spell.h

```diff
@@ -127,6 +127,12 @@ enum SpellRangeFlag
     SPELL_RANGE_RANGED              = 2      //hunter range and ranged weapon
 };
 
+enum SpellMinLevelBlackList : unsigned int32
+{
+    SPELL_NO_MIN_LEVEL_BEAST_LORE = 1462 // Beast Lore Spell ID
+        
```

Comment:
Okay, I am new to git , but I think I fixed it. So I need to remove this pull and make a new one? or how does this work?